src/components/Categories.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const Categories = () => (
+  <div>
+    <h2>Under construction</h2>
+  </div>
```

Comment:
- [ ] According to the project specifications this components should display only a button with text "Check status". Kindly fix this.